content/departments/engineering/dev/process/releases/release_issue_template.md

```diff
@@ -31,6 +31,7 @@ Perform these steps three days before the release date to generate a stable rele
   ```sh
   yarn release release:status
   ```
+- [ ] Create a `Security release approval issue` and post a message in the [#security](https://sourcegraph.slack.com/archives/C1JH2BEHZ) channel tagging @security-support.
```

Comment:
Is this different than the 5 day prior security image scan that security is doing?

Reply:
The scans happen nightly. We need to cross-check the last scan and create a table with accepted CVEs and justifications such as https://handbook.sourcegraph.com/departments/security/tooling/trivy/3-43-1/.

It's a need for strategic customers that we provide them these justifications along with the release. We will look into automating portions of this from the our side.

Reply:
We already have a 5-day prior event for minor releases, and based on the discussion on Slack, I think this check item should be actually added to the patch release template instead? https://github.com/sourcegraph/handbook/blob/andre/release-process-update/content/departments/engineering/dev/process/releases/patch_release_issue_template.md

Reply:
I missed the patch release template, we want this in place for both. The event is helpful but doesn't serve as a gate for the release.

Reply:
@unknwon added it to patch releases as well.